README.md

```diff
@@ -1,71 +1,354 @@
+| `NginxIngressLatencyTooHigh`        | This alert fires if the ingress 99th percentile latency was more than 10 seconds in the last 10 minutes.                                            | critical |   10m    |
+| `NginxIngressCertificateExpiration` | This alert fires if the certificate for a given host is expiring in less than 7 days.                                                               | warning  |          |
+| `NginxIngressCertificateExpiration` | This alert fires if the certificate for a given host is expiring in less than 1 day.                                                                | critical |          |
+
+### Deployment
+
+1. Once you selected the type of ingress you want to deploy (`nginx`, `dual-nginx` or `nginx-gke`) the next step is to specify this in a `Furyfile.yml`:
+
+Single Ingress:
+
+```yaml
+bases:
+  - name: ingress/nginx
+    version: "v1.12.0"
+```
+
+Dual Ingress:
+
+> `dual-nginx` depends on the `nginx` package, so we need to download both of them.
+
+```yaml
+bases:
+  - name: ingress/nginx
+    version: "v1.12.0"
+  - name: ingress/dual-nginx
+    version: "v1.12.0"
+```
+
+GKE:
+
+> `nginx-gke` depends on the `nginx` package, so we need to download both of them.
+
+```yaml
+bases:
+  - name: ingress/nginx
+    version: "v1.12.0"
+  - name: ingress/nginx-gke
+    version: "v1.12.0"
+```
+
+OVH:
+
+> `nginx-ovh` depends on the `nginx` package, so we need to download both of them.
+
+```yaml
+bases:
+  - name: ingress/nginx
+    version: "v1.12.0"
+  - name: ingress/nginx-ovh
+    version: "v1.12.0"
+```
+
+> See `furyctl` [documentation][furyctl-repo] for additional details about `Furyfile.yml` format.
+
+2. Execute `furyctl vendor -H` to download the packages
+
+3. Inspect the download packages under `./vendor/katalog/ingress/`.
+
+4. Define a `kustomization.yaml` that includes the `./vendor/katalog/ingress` directory as resource.
+
+```yaml
+resources:
+- ./vendor/katalog/ingress
+```
+
+5. Apply the necessary patches. You can find a list of common customization [here](#common-customizations).
+
+6. To deploy the packages to your cluster, execute:
+
+```bash
+kustomize build . | kubectl apply -f -
+```
+
+Your are now ready to expose your applications using Kubernetes `Ingress` objects.
+
+### Common customizations
+
+`nginx` package is deployed by default as a `DaemonSet`, meaning that it will deploy 1 ingress-controller pod on every worker node.
+
+This is probably NOT what you want, standard Fury clusters have at least 1 `infra` node (nodes that are dedicated to run Fury infrastructural components, like Prometheus, elasticsearch, and the ingress controllers).
+
+If your cluster has `infra` nodes you should patch the daemonset adding the `NodeSelector` for the `infra` nodes to the Ingress `DaemonSet`. You can do this usiing the following kustomize patch:
```

Comment:
For us, it's obvious what `infra` nodes are but people won't necessarily know. Without going into the rabbit hole of explaining the reference architecture maybe a small clarification could help.

Reply:
isn't the clarification between `()` in line 175 enough?

Reply:
yes that's enough. Missed that 🤦🏻